dask/dataframe/core.py

```diff
@@ -6465,3 +6465,19 @@ def series_map(base_series, map_series):
     divisions = list(base_series.divisions)
 
     return new_dd_object(graph, final_prefix, meta, divisions)
+
+
+categoricalDtype_dispatch = Dispatch("categoricalDtype")
+
+
+def categoricalDtype(meta, categories=None, ordered=False):
+    func = categoricalDtype_dispatch.dispatch(type(meta))
+    return func(categories=categories, ordered=ordered)
+
+
+@categoricalDtype_dispatch.register((pd.DataFrame, pd.Series, pd.Index))
+def categoricalDtype_pandas(categories=None, ordered=False):
+    if PANDAS_VERSION > "0.23.4":
+        return pd.CategoricalDtype(categories=categories, ordered=ordered)
+    else:
+        return pd.api.types.CategoricalDtype(categories=categories, ordered=ordered)
```

Comment:
Can you move this to `dataframe/categorical.py`?

Reply:
Move this dispatch to `categorical.py`